app/api/api_v1/endpoints/assignment_router.py

```diff
@@ -89,13 +102,41 @@ async def grade_assignment(
     request: Request,
     db: Session = Depends(get_db),
     assignment_name: str,
-    grading_body: GradingBody,
+    grading_body: OtterGradingBody,
     perm: None = Depends(PermissionDependency(UserIsInstructorPermission))
 ):
     assignment = await AssignmentService(db).get_assignment_by_name(assignment_name)
-    grade_report = await GradingService(db).grade_assignment(
+    return await GradingService(db).grade_assignment(
         assignment,
         grading_body.master_notebook_content,
         grading_body.otter_config_content
     )
-    return grade_report
\ No newline at end of file
+
+@router.post(
+    "/assignments/{assignment_name}/grade_manual",
+    response_model=GradeReportSchema
+)
+async def grade_assignment(
```

Comment:
This should be named `grade_assignment_manually()` not just `grade_assignment()` since the endpoint is `.../grade_manual`